sklearn/preprocessing/discretization.py

```diff
@@ -8,12 +11,14 @@
 import numpy as np
 import warnings
 
-from sklearn.base import BaseEstimator, TransformerMixin
-from sklearn.preprocessing import OneHotEncoder
-from sklearn.preprocessing.data import _transform_selected
-from sklearn.utils.validation import check_array
-from sklearn.utils.validation import check_is_fitted
-from sklearn.utils.validation import column_or_1d
+from . import OneHotEncoder
+from .data import _transform_selected
+
+from ..base import BaseEstimator, TransformerMixin
+from ..utils.validation import check_array
+from ..utils.validation import check_is_fitted
+from ..utils.validation import column_or_1d
+from ..utils.fixes import np_version
 
 
 class KBinsDiscretizer(BaseEstimator, TransformerMixin):
```

Comment:
Need to update the doc below:
``Bins continuous data into k equal width intervals.`` is no longer the case I think.

Reply:
Thanks